docs/outputs.rst

```diff
@@ -203,8 +245,50 @@ An example of these plots follows:
     in cortical gray matter regions (blue), subcortical gray matter (orange),
     cerebellum (green) and the union of white-matter and CSF compartments (red).
 
+Noise components computed during each CompCor decomposition are evaluated according
+to the fraction of variance that they explain across the nuisance ROI.
+This is used by fMRIPrep to determine whether each component should be saved for
+use in denoising operations: a component is saved if it contributes to explaining
+the top 50 percent of variance in the nuisance ROI. fMRIPrep can be configured to
+save all components instead using the command line option ``--return-all-components``.
+FMRIPrep reports include a plot of the cumulative variance explained by each
+component, ordered by descending singular value.
+
+.. figure:: _static/sub-01_task-rest_compcor.svg
+    :scale: 100%
+
+    The figure displays the cumulative variance explained by components for each
+    of four CompCor decompositions (left to right: anatomical CSF mask, anatomical
+    white matter mask, anatomical combined mask, temporal). The number of
+    components is plotted on the abscissa and the cumulative variance explained
+    on the ordinate. Dotted lines indicate the minimum number of components
+    necessary to explain 50%, 70%, and 90% of variance in the nuisance mask. By
+    default, only the components that explain the top 50% of variance are saved.
```

Comment:
```suggestion
    By default, only the components that explain the top 50% of the variance
    are saved.
```

Reply:
Is it possible to change this threshold of 50% with some flag or some other mechanism?

Reply:
Not possible right now, but it wouldn't take much effort to change the `--return-all-components` option to `--compcor-variance-threshold` with this behaviour.